git-guides/git-status.md

```diff
@@ -0,0 +1,41 @@
+## General Specifications
+
+# Git Status
+
+```sh
+git status
+```
+
+`git status` shows the current state of your Git working directory and staging area.
+
+<!--*   Spiderweb menu of related terms here. What does this mean?-->
+
+## What Does `git status` Do?
+
+When in doubt, run `git status`. This is _always_ a good idea. The `git status` command only outputs information, it won't modify commits or changes in your local repository.
+
+A useful feature of `git status` is that it will provide helpful information depending on your current situation. In general, you can count on it to tell you:
+
+- Where `HEAD` is pointing, whether that is a branch or a commit (this is where you are "checked out" to)
+- If you have any changed files in your current directory that have not yet been committed
+- If changed files are un-staged or staged
```

Comment:
```suggestion
- If changed files are staged or not
```

This more closely matches Git's terminology for unstaged changes: `Changes not staged for commit`